Screen-Sharing-Video-Chat /js/app.js

```diff
@@ -0,0 +1,123 @@
+/* global OT API_KEY TOKEN SESSION_ID SAMPLE_SERVER_BASE_URL */
+
+let apiKey;
+let sessionId;
+let token;
+let screenPublisher;
+let session;
+let publisher;
+let canPublish = false;
+// Handling all of our errors here by alerting them
+function handleError(error) {
+  if (error) {
+    alert(error.message);
+  }
+}
+
+
+function toggleScreen() {
+  const screenSharing = document.getElementById('screen-sharing');
+  // If the screen publisher is connected to the session, unpublish screen and re-publish camera footage
```

Comment:
Let's not stop publishing the camera stream when you publish the screen-sharing stream. That just complicates things, and it's not necessary to illustrate how to publish a screen-sharing stream.